scripts/tick-cluster.js

```diff
@@ -54,6 +54,16 @@ function formatDate() {
     return lpad(now.getHours(), 2) + ':' + lpad(now.getMinutes(), 2) + ':' + lpad(now.getSeconds(), 2) + '.' + lpad(now.getMilliseconds(), 3);
 }
 
+function shuffle(a) {
```

Comment:
One of the dependencies (`underscore`) already implements a `shuffle`-function. It would make more sense to use that one instead of implementing our own.